File: DarkOrbit 10.0/Net/netty/commands/ClientUISlotBarCategoryItemTimerStateModule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ow.Utils;

namespace Ow.Net.netty.commands
{
    class ClientUISlotBarCategoryItemTimerStateModule
    {
        public static short ID = 686;

        public static short READY = 0;
        public static short ACTIVE = 1;
        public static short short_2168 = 2;

        public short var_1238 = 0;

        public ClientUISlotBarCategoryItemTimerStateModule(short param1)
        {
            this.var_1238 = param1;
        }

        public byte[] write()
        {
            var param1 = new ByteArray(ID);
            param1.writeShort(637);
            param1.writeShort(this.var_1238);
            return param1.Message.ToArray();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/serverCommands/ClientUISlotBarCategoryItemTimerStateModule.java
package simulator.netty.serverCommands;

import java.io.DataOutputStream;
import java.io.IOException;

import simulator.netty.ServerCommand;


public class ClientUISlotBarCategoryItemTimerStateModule
        implements ServerCommand {

    public static int   ID         = 29841;
    public static short READY      = 0;
    public static short ACTIVE     = 1;
    public static short short_2168 = 2;

    public short var_1238 = 0;

    public ClientUISlotBarCategoryItemTimerStateModule(short param1) {
        this.var_1238 = param1;
    }

    public int getID() {
        return ID;
    }

    public int method_1005() {
        return 0;
    }

    public void write(DataOutputStream out) {
        try {
            out.writeShort(ID);
            this.writeInternal(out);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }

    protected void writeInternal(DataOutputStream out) {
        try {
            out.writeShort(-10628);
            out.writeShort(this.var_1238);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}